src/Sylius/Behat/Context/Ui/Admin/ManagingPaymentMethodsContext.php

```diff
@@ -459,6 +470,20 @@ public function iConfigureItWithTestStripeGatewayData()
         $this->createPage->setStripePublishableKey('TEST');
     }
 
+    /**
+     * @When I configure it with only :element
+     */
+    public function iConfigureItWithOnly(string $element): void
+    {
+        if ($element === 'secret key') {
+            $this->createPage->setStripeSecretKey('TEST');
+        }
+
+        if ($element === 'publishable key') {
+            $this->createPage->setStripePublishableKey('TEST');
+        }
```

Comment:
Maybe use a match?

Reply:
Or
```suggestion
        if ($element === 'secret key') {
            $this->createPage->setStripeSecretKey('TEST');
            return;
        }

        $this->createPage->setStripePublishableKey('TEST');
```

If we want to protect against passing anything else, there should be a `throw` at the end + `match/switch` statement as Janek said.